airflow/www/views.py

```diff
@@ -105,7 +105,7 @@
 from airflow.models.dag import get_dataset_triggered_next_run_info
 from airflow.models.dagrun import RUN_ID_REGEX, DagRun, DagRunType
 from airflow.models.dataset import DagScheduleDatasetReference, DatasetDagRunQueue, DatasetEvent, DatasetModel
-from airflow.models.errors import ParseImportError
+from airflow.models.errors import ImportError
```

Comment:
Is this an artefact from previous changes or why do you change the Error class in views.py in this PR now?

Reply:
Something must have modified an earlier version of the code, and I've reverted that part back to the original.